core-java-modules/core-java-arrays-guides/src/test/java/com/baeldung/print2darrays/Print2DArraysUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.print2darrays;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
```

Comment:
Missing:
import org.junit.jupiter.api.extension.ExtendWith;

This code never compiled, and even after fixing this, the tests do not pass. This is the last review I will be doing on a broken build. Please compile and run the tests before submitting it for review.

Reply:
code compiled successfully.